zerver/tests/tests.py

```diff
@@ -2132,7 +2132,7 @@ def test_open_realm_logic(self):
         mit_realm.save()
         self.assertTrue(completely_open(mit_realm.domain))
         self.assertEquals(get_unique_open_realm(), None)
-        with self.settings(SYSTEM_ONLY_REALMS={"zulip.com"}):
+        with self.settings(SYSTEM_ONLY_REALMS={"zulip.com", "analytics.test"}):
```

Comment:
Perhaps we should just add a tearDown method to the analytics class to remove that realm when the tests that create it finish running?